tests/integration/test_render.py

```diff
@@ -213,6 +214,30 @@ def test_popup(self):
         self.assertEqual(popup_content.text, "")
         self.assertEqual(popup_wrapper.get_attribute('aria-live'), 'polite')
 
+    @data(None, Keys.RETURN)
+    def test_go_to_beginning_button(self, action_key):
+        self.load_scenario()
+        self.scroll_down(250)
+
+        button = self._get_go_to_beginning_button()
+        self.assertEqual(button.get_attribute('tabindex'), '0')
+        # Button is only visible to screen reader users by default.
+        self.assertIn('sr', button.get_attribute('class').split())
+        # Set focus to the element (cannot find a way to do this without execute_script).
+        self.browser.execute_script('$("button.go-to-beginning-button").focus()')
```

Comment:
I tried that initially, but it didn't work. I did some more digging and found out why it does not work - when you TAB to the element in firefox, the `focus` event is not fired unless Firefox is the focused window, which it usually isn't when running the tests. I expanded the comment to explain this.